packages/cli/src/commands/new/entity.ts

```diff
@@ -15,6 +15,7 @@ import * as path from 'path'
 import { generate } from '../../services/generator'
 import { templates } from '../../templates'
 import { checkItIsABoosterProject } from '../../services/project-checker'
+import inflection = require('inflection')
```

Comment:
I think that you can avoid the call to `require` doing this:

```suggestion
import * as inflection from 'inflection'
```

Reply:
Thanks! Fixed here: https://github.com/boostercloud/booster/blob/5069a978f11be9c41c0602e261b634a7c0e25a2f/packages/cli/src/commands/new/entity.ts#L18